README.md

```diff
@@ -71,10 +77,17 @@ Icons
 The icons should be 16x16 pixels. I recommend the [Fugue icon set](http://p.yusukekamiyamane.com/) (shadowless).
 
 
-kthxbye
 
--[joe](http://joewalnes.com)
+CHANGES FROM ORIGINAL
```

Comment:
"Changes from original" does not make sense to end users if this pull request is merged.